Backpack/Card/Classes/BPKCard.h

```diff
@@ -34,6 +36,13 @@ IB_DESIGNABLE @interface BPKCard : UIControl
  */
 @property(nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
 
+/**
+ * Determines how the card should appear to assistive technology.
+ * Default is BPKCardAccessibilityConfigurationContainer.
+ * @see BPKCardAccessibilityConfiguration
+ */
+@property(strong, nonatomic, nonnull) BPKCardAccessibilityConfiguration *accessibilityConfiguration;
```

Comment:
Should we call this just `configuration`? It's encapsulates more than just accessibility, right?

Reply:
Right now it doesn't - it's literally just 3 accessibility properties.
Can make it more generically named if you want though 👍

Reply:
WDYT?

Reply:
I don't think a more generic name is deterimental and it would be more future proof 🤔